### PR TITLE
🐛 MT-1118: make voiceover annonce characters remaining

### PR DESCRIPTION
**Pull request by itemodahl**
* If more characters are entered, let user know how many characters exceeds the limit.

MT-1118

### CODE REVIEW

**Review by lha047 — CHANGES_REQUESTED** [review]
I cant build the web project with this version of DS.

`'countCharactersTooManyLabel' is not exported by node_modules/@mattilsynet/designsystem/src/svelte/form/TextArea.svelte, imported by node_modules/@mattilsynet/designsystem/src/svelte/form/TextInput.svelte
`

**Comment by amundlindstol** [comment]
skal versjonen bumpes?

**Comment by itemodahl** [comment]
@amundlindstol tenker du på når du tester?

File: src/svelte/form/TextInput.svelte [hunk 1]
```diff
@@ -24,6 +25,15 @@
 
   export let inputClass = ''
   export let isHorizontal = false
+  console.log('maxlength %s', JSON.stringify(maxlength, null, 2))
```
[the remark this answers is not in the document]
Comment (itemodahl) [reply src/svelte/form/TextInput.svelte]: Removed

**Review by lha047 — APPROVED** [review]
Nice :D Bra jobba!

**Pull request merged**